sticky_notes.py

```diff
@@ -81,3 +81,37 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_note(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) < 2:
+            return "Missing note number argument."
+
+        if len(args) >= 2:
+            nick = args[0]
+            note_num = args[1]
+
+            if nick in self.notes:
+                notes = self.notes[nick]
+                print len(self.notes[nick])
+                to_pop = -2
```

Comment:
@JakubNvk you already did on line 109 by testing `note_num.isdigit()`. Anything that's not a digit will not pass that test which includes `-1` or `-5`.